src/App.tsx

```diff
@@ -1,8 +1,43 @@
+import { useState } from "react";
+
+import { DisplayCard } from "./components/DisplayCard/DisplayCard";
+
 const App = () => {
+	const [nameInputValue, setNameInputValue] = useState<string>("");
+	const [entities, setEntities] = useState<any[]>([]);
+
+	const queryString = nameInputValue.toLowerCase();
+	const fetchURL = `https://pokeapi.co/api/v2/pokemon/${queryString}/`;
```

Comment:
Should probably create a separate file to handle the API calls to Poke API,  so they can share common URLs like this and not messy up the individual components.